hi-ml/src/health_ml/deep_learning_config.py

```diff
@@ -153,18 +155,24 @@ class WorkflowParams(param.Parameterized):
                      doc="When comparing CSV files during regression tests, use this value as the maximum allowed "
                          "relative difference of actual and expected results. Default: 0.0 (must match exactly)")
     regression_metrics: str = param.String(default=None, doc="A list of names of metrics to compare")
+    run_inference_only: bool = param.Boolean(False, doc="If True, run inference using the specified ckpt_run_id as "
```

Comment:
What about the weights_url or local_weights_path?

Reply:
Good point, will update the doc.